### PR TITLE
Email Spoofing to Inbox due to Missing or Misconfigured DMARC on Emai…

### PR DESCRIPTION
**Pull request by RRudder**
…l Domain

### CODE REVIEW

File: submissions/description/server_security_misconfiguration/mail_server_misconfiguration/email_spoofing_to_inbox_due_to_missing_or_misconfigured_dmarc_on_email_domain/template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Steps to Reproduce
+
+1. Use dig or nslookup to request details for DMARC:
+
+```bash
+dig TXT 
+```
+
+1. Observe in the response that there is no answer section for DMARC on the domain:
```
Comment (evildaemond) [inline submissions/description/server_security_misconfiguration/mail_server_misconfiguration/email_spoofing_to_inbox_due_to_missing_or_misconfigured_dmarc_on_email_domain/template.md:19]: Small technical wording change

1. Observe in the response that there is a misconfigured or missing answer section for DMARC on the domain:

**Review by evildaemond — CHANGES_REQUESTED** [review]
1 quick change then LGTM

File: submissions/description/server_security_misconfiguration/mail_server_misconfiguration/email_spoofing_to_inbox_due_to_missing_or_misconfigured_dmarc_on_email_domain/template.md (hunk 1 above)
Comment (RRudder) [reply submissions/description/server_security_misconfiguration/mail_server_misconfiguration/email_spoofing_to_inbox_due_to_missing_or_misconfigured_dmarc_on_email_domain/template.md:19]: Thanks again for the tech review!

**Review by evildaemond — APPROVED** [review]

**Comment by evildaemond** [comment]
LGTM

**Pull request merged**